app/views/stats/subscriptions.html.erb

```diff
@@ -2,6 +2,25 @@
     <h4> About this page </h4>
     <p> People subscribe to tags so that they can be updated when a research is posted with the tag they are subscribed to. This page shows the number of subscriptions and subscription rates on different tags. </p>
 </div>
+
 <div class="col-md-8 col-md-offset-1">
     <h2>Subscriptions</h2>
-</div>
+ 	<table class="table inline-grid">
+		<tr>
+			<thead>
+				<th>Subscriptions</th>
+				<th>Tags</th>
+			</thead>
+		</tr>
+		<% @tags.each do |range,tags| %>
+		<tr>
+			<tbody>
+				<td><%= range * 10 %> - <%= range * 10 + 9 %></td>
+				<% tags.each do |tag| %>
+				<td><%= tag[0]%>:<%=tag[1] %></td>
```

Comment:
Oh, aha - yes, so instead of the `<td>` we need to also use `<p>` tags - and the `<td>` tags should go outside the `each` loop. That way the whole collection will be in a single table cell, but each tag will be in its own paragraph (on its own line). Make sense?

Reply:
Yes though long  it's clear. and more readable too. 

![ptags](https://user-images.githubusercontent.com/17080976/54864021-82045800-4d61-11e9-9eda-fffc86b65605.gif)

Reply:
Great! Actually let's tackle a few more things in a follow-up PR:

1. let's sort this DESCENDING or `desc` so we start with the most-used tags
2. let's hide anything more than 25 results and show a "View all" link to show them?
3. let's make the tag names into links that go to `/tag/TAGNAME`

and we can do more! This is a great start!